src/creature.h

```diff
@@ -544,6 +544,9 @@ class Creature
         virtual const std::string &symbol() const = 0;
         virtual bool is_symbol_highlighted() const;
 
+		// Turn since creature was milked for last time
+		int turn_next_milking = 0;
```

Comment:
As this only every applies to milked monsters (which would be *very* few), it might be better to make this an effect. Something like "was_milked" that times out after a day.

Also: the member is currently not saved at all. Saving and loading a game will just reset this to 0. Effects are automatically saved.